src/helpers/xml-builder.js

```diff
@@ -75,6 +75,25 @@ const fixupColorCode = (colorCodeString) => {
   }
 };
 
+const cssBorderParser = (borderString) => {
+  let [size, stroke, color] = borderString.split(' ');
+
+  if (pointRegex.test(size)) {
+    const matchedParts = size.match(pointRegex);
+    // convert point to eighth of a point
+    size = pointToEIP(matchedParts[1]);
+  } else if (pixelRegex.test(size)) {
+    const matchedParts = size.match(pixelRegex);
+    // convert pixels to eighth of a point
+    size = pixelToEIP(matchedParts[1]);
+  }
+  stroke = stroke && ['dashed', 'dotted', 'double'].includes(stroke) ? stroke : 'single';
+
+  color = color && fixupColorCode(color).toUpperCase();
+
+  return [size, stroke, color];
+};
+
```

Comment:
@amrita-syn isn't it better to make use of the eslint rule disable feature rather than moving the code around?
`// eslint-disable-next-line no-use-before-define`

Reply:
@privateOmega , The method is referenced 5 times before definition and only once after. Do you still want me to use the lint ignore option?

Reply:
@amrita-syn tbh I do feel that and the reason is to preserve git history

Reply:
@privateOmega Done.